exposee-app/server.js

```diff
@@ -65,7 +113,21 @@ app.post('/videos', async (req, res) => {
   res.status(500).json({ message: error.message });
   }
 });
-
+app.post('/broadcast', async (req, res)=> {
+  const userId = req.body.id;
+  try{
+    const user = await User.findByPk(userId);
+    if(!user){
+      return res.status(404).json({message: 'User not found'});
+    }
+    user.isLive = true;
+    await user.save();
+    res.json({message: 'You are live'});
+  }catch (error){
```

Comment:
let's clarify what the broadcast endpoint should be doing ? I thought it would be creating a video record in the video table